### DISCUSSION TITLE
[Feature] Sentry mode / Vampire drain Map overview

### DISCUSSION THREAD

**Discussion by ogamingSCV** [teslamate-org/teslamate#3996] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

I love the Charge map with the bubbles which size changes depending on the amount of kWh being charged at the location. 

A nice addition would be to implement a similar Map for Sentry mode / Vampire drain. 
It already records the time and SoC lost while "idle". If this data is being combined with the location data of that time, a similar map could be created to visualize at which location what overall SoC is lost. 

With this feature the user could easily determine if it might be good to turn Sentry mode off at specific location (with exclude favorites etc.) or just get a better sense of the vampire drain. 

The combined data provides the ability to overview the lost SoC, amount, idle stops and avg. of idle time at a given point on the map. 

### Relevant log output

```shell
-
```

### Type of installation

Docker

### Version

1.29.2

**Comment by cwanja** [comment]
Converting to discussion under **Ideas**. That is where we track feature requests. Issues are for current features that are classified as bugs. Thanks for the suggestion! We always encourage individuals to create a pull request as well.

**Comment by cwanja** [comment]
@jheredianet @DrMichael dashboard ideas.

**Comment by rywards** [comment]
If you're good with postgres you could create your own dashboard with the data. I'm planning on putting something together like that on my own instance.

**Reply by jheredianet** [reply]
If you do, we encourage you to share it through a pull request, then we can all contribute and improve if possible.
